test/DataCollectors/Microsoft.TestPlatform.Extensions.EventLogCollector.UnitTests/EventLogSessionContextTests.cs

```diff
@@ -70,18 +67,18 @@ public void CreateEventLogContainerShouldCreateNegativeEndIndexIfLogEntriesAreEm
     }
 }
 
-public class DummyEventLogContainer : IEventLogContainer
+public sealed class DummyEventLogContainer : IEventLogContainer
 {
     public DummyEventLogContainer(bool initialize)
     {
         EventLogEntries = new List<EventLogEntry>(10);
-        EventLog eventLog = new("Application");
+        EventLog = new("Application");
```

Comment:
We were creating a local variable but never assigning the property. I assumed this was an error and the tests still don't fail (locally) so I guess this is the right move.